--- origsrc/MediaInfo/Project/GNU/CLI/Makefile.am	2012-05-11 02:49:42.000000000 -0500
+++ src/MediaInfo/Project/GNU/CLI/Makefile.am	2012-05-31 23:10:24.131998300 -0500
@@ -9,4 +9,4 @@ mediainfo_SOURCES = \
 
 #mediainfo_LDFLAGS     = -no-undefined -version-info 0:0:0
 
-INCLUDES = -I../../../Source -I../../../../MediaInfoLib/Source -I../../../../ZenLib/Source
\ No newline at end of file
+INCLUDES = -I$(srcdir)/../../../Source -I$(srcdir)/../../../../MediaInfoLib/Source -I$(srcdir)/../../../../ZenLib/Source
--- origsrc/MediaInfo/Project/GNU/GUI/Makefile.am	2012-05-11 02:49:42.000000000 -0500
+++ src/MediaInfo/Project/GNU/GUI/Makefile.am	2012-05-31 23:27:05.967215200 -0500
@@ -22,4 +22,4 @@ mediainfo_gui_SOURCES = \
 
 #mediainfo-gui_LDFLAGS     = -no-undefined -version-info 0:0:0
 
-INCLUDES = -I../../../Source -I../../../../MediaInfoLib/Source -I../../../../ZenLib/Source
\ No newline at end of file
+INCLUDES = -I$(srcdir)/../../../Source -I$(srcdir)/../../../../MediaInfoLib/Source -I$(srcdir)/../../../../ZenLib/Source
--- origsrc/MediaInfoLib/Project/GNU/Library/Makefile.am	2012-05-11 02:50:33.000000000 -0500
+++ src/MediaInfoLib/Project/GNU/Library/Makefile.am	2012-05-31 22:46:56.868973900 -0500
@@ -179,4 +179,4 @@ endif
 
 libmediainfo_la_LDFLAGS     = -no-undefined -version-info 0:0:0
 
-INCLUDES = -I../../../Source -I../../../../ZenLib/Source
+INCLUDES = -I$(srcdir)/../../../Source -I$(srcdir)/../../../../ZenLib/Source
--- origsrc/MediaInfoLib/Project/GNU/Library/configure.ac	2013-05-11 16:37:39.000000000 -0500
+++ src/MediaInfoLib/Project/GNU/Library/configure.ac	2013-06-19 03:39:27.323190800 -0500
@@ -564,8 +564,8 @@ if test "$with_libmd5" = "yes"; then
 	AM_CONDITIONAL(COMPILE_MD5, false)
 	using_libmd5="system"
 else
-	CXXFLAGS="$CXXFLAGS -I../../../Source/ThirdParty/md5"
-	CFLAGS="$CFLAGS -I../../../Source/ThirdParty/md5"
+	CXXFLAGS="$CXXFLAGS -I$srcdir/../../../Source/ThirdParty/md5"
+	CFLAGS="$CFLAGS -I$srcdir/../../../Source/ThirdParty/md5"
 	using_libmd5="internal"
 	AM_CONDITIONAL(COMPILE_MD5, true)
 fi
@@ -582,7 +582,7 @@ if test "$with_libtinyxml2" = "yes"; the
 	AM_CONDITIONAL(COMPILE_TINYXML2, false)
 	using_libtinyxml2="system"
 else
-	CXXFLAGS="$CXXFLAGS -I../../../Source/ThirdParty/tinyxml2"
+	CXXFLAGS="$CXXFLAGS -I$srcdir/../../../Source/ThirdParty/tinyxml2"
 	using_libtinyxml2="internal"
 	AM_CONDITIONAL(COMPILE_TINYXML2, true)
 fi
@@ -595,7 +595,7 @@ MediaInfoLib_LibName="$with_libmediainfo
 dnl -------------------------------------------------------------------------
 dnl base64
 dnl
-CXXFLAGS="$CXXFLAGS -I../../../Source/ThirdParty/base64"
+CXXFLAGS="$CXXFLAGS -I$srcdir/../../../Source/ThirdParty/base64"
 
 dnl #########################################################################
 dnl ### Compiler specific
--- origsrc/ZenLib/Project/GNU/Library/Makefile.am	2012-05-18 10:18:52.000000000 -0500
+++ src/ZenLib/Project/GNU/Library/Makefile.am	2012-05-31 22:12:26.558733400 -0500
@@ -79,4 +79,4 @@ bin_SCRIPTS = libzen-config
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libzen.pc
 
-AM_CPPFLAGS = -I../../../Source
+AM_CPPFLAGS = -I$(srcdir)/../../../Source
